README.md

```diff
@@ -21,6 +21,7 @@ A list of of AI coding tools (assistants, completion, refactoring, etc.).
 - [GitLab Code Suggestions](https://docs.gitlab.com/ee/user/project/repository/code_suggestions.html)
 - [Sourcegraph Cody](https://about.sourcegraph.com/cody)
 - [Obsidian copilot auto completion](https://github.com/j0rd1smit/obsidian-copilot-auto-completion)
+- [aiXcoder-7B](https://github.com/aixcoder-plugin/aiXcoder-7B)
```

Comment:
This should be under "Code completion LLMs". I know it has a VS Code and JB plugin but those should be linked directly to the plugin not the LLM